jobbergateapi2/config.py

```diff
@@ -0,0 +1,20 @@
+"""
+Configuration file, sets all the necessary environment variables, it is better used with a .env file
+"""
+from pydantic import BaseSettings, Field
+
+
+class Settings(BaseSettings):
+    DATABASE_URL: str
+    TEST_DATABASE_URL: str
```

Comment:
Don't need to fix this right now but you can put regular expressions on these to help the operator validate.